ag_data/formulas/library/system/mercury_formulas.py

```diff
@@ -0,0 +1,50 @@
+#
+# Formulas for Mercury's built-in and natively supported sensor types
+# Sample processing formulas with primary sensors and fictional Mercury-branded sensors
+#
+
+from ag_data import models
+
+
+def fPass(timestamp, sensor, measurement_payload):
+    result = {}
+
+    return result
+
+
+def fMercurySimpleTemperatureSensor(timestamp, sensor, measurement_payload):
+    result = {}
+
```

Comment:
It would be great we leave a comment like `# TODO: blah blah`. Or even better we could have
```
assert False, "not implemented"
```

Reply:
Oh yes, this is a simple temperature sensor that really does nothing. I should add a comment to explain that it does nothing. Or we could also come up with something for it to do, like calculate and save a representation in Fahrenheit or something. (sorry this is a bad example, because storing unit converted value as a separate field is so wasteful)

Reply:
Hmm if the implementation is intentional and finished, how about just aliasing fEmptyResult to fMercurySimpleTemperatureSensor by just putting:
```
fMercurySimpleTemperatureSensor = fEmtpyResult
```

Reply:
What do you think of this: https://github.com/jackxujh/spring2020-cs-gy-9223-class/blob/1bd0edf582ec4bae50a266dab15bb36b99ae069c/ag_data/formulas/library/system/mercury_formulas.py#L14-L19

Reply:
This is now in #376.